ui/imports/shared/popups/walletconnect/DAppRequestModal.qml

```diff
@@ -0,0 +1,286 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtQml.Models 2.14
+import SortFilterProxyModel 0.2
+
+import QtGraphicalEffects 1.15
```

Comment:
The SFPM and Effects imports are needed?

Reply:
Leftovers, will remove it.